### DISCUSSION TITLE
<ask> how to make script rdp windows in codespace

### DISCUSSION THREAD

**Discussion by sadewa999** [community/community#42233] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Help me to script visual code..

**Comment by cmbrose** [comment]
👋 Thanks for reaching out @sadewa999! However, I don't quite understand what you're trying to accomplish here - can you explain a bit more?

**Reply by sadewa999** [reply]
ok @cmbrose,, usually me, deploy via github to make VPS with ip ngrok,, 

My script deploy error not tuning to putty,,

**Reply by jaysee260** [reply]
Hi @sadewa999 👋 . Sorry about the confusion, but the nature of the error you mention is still not super clear to me. Is the error with your deploy script because of a possible configuration error? Or is it a problem with the codespace?

Do you mind providing some more details about the error itself?

**Comment by Musthuakqha** [comment]
Hi

**Reply by sadewa999** [reply]
Hi too @Musthuakqha

**Comment by mishra-prajjawal** [comment]
Hello 👋 @sadewa999 Thank you for reaching out. Now to your question I don't think that is allowed on codespaces so don't try it as you may end up losing your account in that process.

Lastly If you need a VPS or VM for your own business then you may get one from a cloud provider like Microsoft Azure or Amazon AWS or Google Cloud Platform.

I hope I answered your questions.

**Comment by jfullstackdev** [comment]
maybe you mean collaborate with VS Code dev ? 

here it is, it's on GitHub too 

https://github.com/microsoft/vscode
